service/platform/index.js

```diff
@@ -207,11 +207,14 @@ class ServerPlatform extends Tp.BasePlatform {
                     hasFilterHeuristics = true;
                 if (mod.name === 'module-filter-apply')
                     hasFilterApply = true;
+                if (mod.name === 'module-role-ducking')
+                    await this._pulse.unloadModule(mod.index);
```

Comment:
Do we get the list of parameters here? Should we check that this is the right instance of module-role-ducking, in case the user has a custom PulseAudio configuration for other reasons?

Reply:
We do have the list but we are not checking the arguments given that only a single instance of module-role-ducking can be loaded.

Reply:
Oh ok then